esp-hal-common/ld/sections/external.x

```diff
@@ -0,0 +1,36 @@
+
+
+SECTIONS {
+  .external.data :
```

Comment:
To support `.data` in ext-ram we would need a way to copy the initial data here

Reply:
Yes, I know.  I just want to first understand if I took everything into account for bss, at least 😅